### PR TITLE
Enh merge 0.11.x to master

### PR DESCRIPTION
**Pull request by yarikoptic**
to bring in
      NF: rerun: Add --explicit flag
      BF: annexrepo: Adjust overly selective is_special_annex_remote()

### CODE REVIEW

File: datalad/interface/tests/test_rerun.py [hunk 1]
```diff
@@ -757,6 +758,158 @@ def test_run_inputs_no_annex_repo(path):
     ds.rerun()
 
 
+@slow  # ~10s
+@known_failure_windows
+@with_testrepos('basic_annex', flavors=['clone'])
+def test_run_explicit(path):
```
Comment (kyleam) [inline datalad/interface/tests/test_rerun.py]: This is wrong for two reasons: you introduce test_run_explicit *twice* in merge, and test_run_explicit is already included in core/local/tests/test_run.py.

File: datalad/interface/tests/test_rerun.py [hunk 2]
```diff
@@ -757,6 +758,158 @@ def test_run_inputs_no_annex_repo(path):
...
+
+    # Saving explicit outputs works from subdirectories.
+    subdir = opj(path, "subdir")
+    mkdir(subdir)
+    with chpwd(subdir):
+        run("echo insubdir >foo", explicit=True, outputs=["foo"])
+    ok_(ds.repo.file_has_content(opj("subdir", "foo")))
+
+
+@known_failure_windows
```
Comment (kyleam) [inline datalad/interface/tests/test_rerun.py]: From here on is fine.

File: datalad/interface/tests/test_rerun.py (hunk 1 above)
Comment (yarikoptic) [reply datalad/interface/tests/test_rerun.py]: Oops, sorry. May be you could just redo this PR? I will not get to the laptop for a bit

Comment (kyleam) [reply datalad/interface/tests/test_rerun.py]: I've updated your branch.

**Comment by yarikoptic** [comment]
Thank you @kyleam !  merging!

**Pull request merged**